_posts/2018-08-28-announcing-riff-0-1-2-on-Knative.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: single
+title: "Announcing riff v0.1.2 on Knative"
+header:
+  overlay_image: /images/liverpool-st.jpg
+excerpt:
+  With verbose builds and function chaining using Channels and Subscriptions    
+permalink: /blog/announcing-riff-0-1-2-on-Knative/
+---
+
+[riff v0.1.2](https://github.com/projectriff/riff/releases/tag/v0.1.2) on Knative is now available. Many thanks to all riff and Knative contributors.
+
+## install
+We recommend installing riff v0.1.2 on a fresh Kubernetes cluster. The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases) on GitHub. Please follow one of the [getting started](/docs) guides,  to create a new cluster on GKE or minikube.
```

Comment:
Should the link target the 0.1.2 release directly?

Reply:
should we add instructions for uninstalling an older version using `riff system unintstall --istio --force`?

Reply:
good idea - i will add a reminder